database/changesets/20200317_ExternalWholeGenomeReprocessingTable.xml

```diff
@@ -16,15 +16,16 @@
         </comment>
```

Comment:
Does this file really need to be indented for this feature?

Reply:
My suspect is no 🤔